### DISCUSSION TITLE
Why were the patch versions for CVE-2021-21284 and CVE-2021-21285 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [moby/moby#46005] [Q&A] [answered]
Hello, we are a research team working on Golang. During our investigation, we found CVE-2021-21284 was addressed in commit 64bd4485b3a66a597c02c95f5776395e540b2c7c and CVE-2021-21285 was addressed in commit 8d3179546e79065adefa67cc697c09d0ab137d30. However, we noticed that the patch version was released after a year (486 days).  We are curious about the reasons behind the delayed release of the patch version, as it may hinder the efficient distribution of patches to downstream users. Could the reason be 

1.Issues with testing and CI checking. 

2.Other commits have to be incorporated into one release. 

3.By convention, versions are not frequently released. 

4.Other reasons. 

Thank you for your attention, and we look forward to receiving your reply.

**Comment by rumpl** [comment, marked as answer]
As you can see in [this PR](https://github.com/moby/moby/pull/41964), versions [19.03.15](https://github.com/moby/moby/releases/tag/v19.03.15) and [20.10.3](https://github.com/moby/moby/releases/tag/v20.10.3) were released the same day: Feb 2. 2021.

**Comment by Silence-worker-02** [comment]
Thank you for your help.
